src/Libs/OwlCore.WinUI/Converters/DataTriggerBehaviorEnumConverter.cs

```diff
@@ -12,7 +12,7 @@ namespace OwlCore.WinUI.Converters
     public sealed class DataTriggerBehaviorEnumConverter : IValueConverter
     {
         /// <inheritdoc/>
-        public object Convert(object value, Type targetType, object parameter, string language)
+        public object? Convert(object value, Type targetType, object parameter, string language)
         {
             return value.ToString();
```

Comment:
This can break if value is `null` maybe value?.String()?

Reply:
The suggestion doesn't seem related to the changes made for the ticket. We'll handle this when/if it becomes an issue.